src/plugins/lightbox/lightbox.js

```diff
@@ -296,7 +301,7 @@ var componentName = "wb-lbx",
 
 				overlayCloseFtr = "<button type='button' class='btn btn-sm btn-primary pull-left " + closeClassFtr +
 					"' title='" + spanTextFtr + "'>" +
-					closeTextFtr +
+					"<span aria-hidden='true'>" + closeTextFtr + "</span>" +
```

Comment:
```suggestion
					"<span class="wb-inv">" + closeTextFtr + "</span>" +
```

Reply:
Thanks for pointing this @radmorecameron , I didn't realized that it was the close button at the bottom of the overlay. I though it was the top right "X" button.

To make the screen reader less verbose, I recommend to remove the title attribute and to replace the i18n text from "Close overlay" for "overlay" which could end up with this code instead:

```javascript
				overlayCloseFtr = "<button type='button' class='btn btn-sm btn-primary pull-left " + closeClassFtr +
					"'>" +
					closeTextFtr +
					"<span class='wb-inv'>" + spanTextFtr + "</span></button>";
```

Where the i18n text value for `overlay-close` would be modified in this CSV file: https://github.com/wet-boew/wet-boew/blob/master/src/i18n/i18n.csv#L25

Note: If you want to test locally the i18n text change, you will need to modify the following file in your test environment and probably replicate those change for the minified corresponding file:
* https://github.com/wet-boew/wet-boew-dist/blob/v4.0-dist/wet-boew/js/i18n/en.js#L61
* https://github.com/wet-boew/wet-boew-dist/blob/v4.0-dist/wet-boew/js/i18n/fr.js#L61